cmd/argocd-applicationset-controller/commands/applicationset_controller.go

```diff
@@ -113,15 +116,28 @@ func NewCommand() *cobra.Command {
 				os.Exit(1)
 			}
 
+			var cacheOpt ctrlcache.Options
+
+			if watchedNamespace != "" {
+				cacheOpt = ctrlcache.Options{
+					DefaultNamespaces: map[string]ctrlcache.Config{
+						watchedNamespace: {},
+					},
+				}
+			}
+
 			mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-				Scheme:                 scheme,
-				MetricsBindAddress:     metricsAddr,
-				Namespace:              watchedNamespace,
+				Scheme: scheme,
+				Metrics: metricsserver.Options{
+					BindAddress: metricsAddr,
+				},
+				Cache:                  cacheOpt,
 				HealthProbeBindAddress: probeBindAddr,
-				Port:                   9443,
```

Comment:
Is this option intentionally dropped? Does it default to 9443?

Reply:
Yes.
https://github.com/kubernetes-sigs/controller-runtime/blob/479b723944e34ae42c9911fe01228ff34eb5ca81/pkg/webhook/server.go#L39
The port of 9443 became by default